test/ogc-api-coverages/get-coverage-rangeset.ts

```diff
@@ -390,7 +390,7 @@ describe('OGC API Coverages - getCoverageRangeset', function () {
           name: 'turbo',
         },
         steps: [{
-          image: 'fake-internal.earthdata.nasa.gov/nexus-service/foo:uat',
```

Comment:
I believe this was testing something with an internal earthdata.nasa.gov URL. Can we put this back?

Reply:
Yea, this is the thing, if the first step is not query-cmr, things behave differently and the related tests will be broken.
As a matter of fact, I added another test suite [when the first step is giovanni-adapter instead of query-cmr](https://github.com/nasa/harmony/pull/160/files#diff-4265cd083e74e99482c60ed86e35eb2cbddf4db892fd847bb400ebf2fe59192fR480). Maybe we can change the image name there to be `'fake-internal.earthdata.nasa.gov/nexus-service/foo:uat'`?

Reply:
Done ... thanks!